application/blueprints/datamanager/router.py

```
@@ -40,27 +49,83 @@
+
+def _require_login():
     if current_app.config.get("AUTHENTICATION_ON", True):
         if session.get("user") is None:
             return redirect(url_for("auth.login", next=request.url))
 
+
+def _require_add_data_unlocked():
     try:
-        lock = db.session.get(ServiceLock, "add_data")
+        lock = db.session.get(ServiceLock, "add-data")
     except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)